src/socketio.quiver.js

```diff
@@ -410,9 +438,16 @@ QuiverSocialProvider.prototype.connectAsOwner_ = function(server, continuation)
 
   connection.ready.then(function() {
     connection.socket.emit('join', this.configuration_.self.id);
+    // TODO: Sign this message.  Currently, the pgp-e2e module does not support
+    // signing without encryption.  The lack of verification enables a kind of
+    // application-layer reflection/amplification "attack", by forging pings to
+    // a user's friends to trigger intro responses.
```

Comment:
Might be good to comment that the reason we don't encrypt this message is because it is broadcast to multiple clients